.gitignore

```diff
@@ -23,3 +23,4 @@ qt/FileReader/release/
 qt/output/
 
 /node_modules/
+/dist/
```

Comment:
did you mean to include this?

Reply:
yes, that is the destination for the build script when you build the "distribution"